test/functional-tests/FixedPoint.cpp

```diff
@@ -0,0 +1,837 @@
+    <FixedPointParameter Name="q14.1" Size="16" Integral="14" Fractional="1"/>
+    <FixedPointParameter Name="q15.0" Size="16" Integral="15" Fractional="0"/>
+    <FixedPointParameter Name="q0.0" Size="8" Integral="0" Fractional="0"/>
+    <FixedPointParameter Name="q0.1" Size="8" Integral="0" Fractional="1"/>
+    <FixedPointParameter Name="q0.2" Size="8" Integral="0" Fractional="2"/>
+    <FixedPointParameter Name="q0.3" Size="8" Integral="0" Fractional="3"/>
+    <FixedPointParameter Name="q0.4" Size="8" Integral="0" Fractional="4"/>
+    <FixedPointParameter Name="q0.5" Size="8" Integral="0" Fractional="5"/>
+    <FixedPointParameter Name="q0.6" Size="8" Integral="0" Fractional="6"/>
+    <FixedPointParameter Name="q0.7" Size="8" Integral="0" Fractional="7"/>
+    <FixedPointParameter Name="q1.0" Size="8" Integral="1" Fractional="0"/>
+    <FixedPointParameter Name="q1.1" Size="8" Integral="1" Fractional="1"/>
+    <FixedPointParameter Name="q1.2" Size="8" Integral="1" Fractional="2"/>
+    <FixedPointParameter Name="q1.3" Size="8" Integral="1" Fractional="3"/>
+    <FixedPointParameter Name="q1.4" Size="8" Integral="1" Fractional="4"/>
+    <FixedPointParameter Name="q1.5" Size="8" Integral="1" Fractional="5"/>
+    <FixedPointParameter Name="q1.6" Size="8" Integral="1" Fractional="6"/>
+    <FixedPointParameter Name="q2.0" Size="8" Integral="2" Fractional="0"/>
+    <FixedPointParameter Name="q2.1" Size="8" Integral="2" Fractional="1"/>
+    <FixedPointParameter Name="q2.2" Size="8" Integral="2" Fractional="2"/>
+    <FixedPointParameter Name="q2.3" Size="8" Integral="2" Fractional="3"/>
+    <FixedPointParameter Name="q2.4" Size="8" Integral="2" Fractional="4"/>
+    <FixedPointParameter Name="q2.5" Size="8" Integral="2" Fractional="5"/>
+    <FixedPointParameter Name="q3.0" Size="8" Integral="3" Fractional="0"/>
+    <FixedPointParameter Name="q3.1" Size="8" Integral="3" Fractional="1"/>
+    <FixedPointParameter Name="q3.2" Size="8" Integral="3" Fractional="2"/>
+    <FixedPointParameter Name="q3.3" Size="8" Integral="3" Fractional="3"/>
+    <FixedPointParameter Name="q3.4" Size="8" Integral="3" Fractional="4"/>
+    <FixedPointParameter Name="q4.0" Size="8" Integral="4" Fractional="0"/>
+    <FixedPointParameter Name="q4.1" Size="8" Integral="4" Fractional="1"/>
+    <FixedPointParameter Name="q4.2" Size="8" Integral="4" Fractional="2"/>
+    <FixedPointParameter Name="q4.3" Size="8" Integral="4" Fractional="3"/>
+    <FixedPointParameter Name="q5.0" Size="8" Integral="5" Fractional="0"/>
+    <FixedPointParameter Name="q5.1" Size="8" Integral="5" Fractional="1"/>
+    <FixedPointParameter Name="q5.2" Size="8" Integral="5" Fractional="2"/>
+    <FixedPointParameter Name="q6.0" Size="8" Integral="6" Fractional="0"/>
+    <FixedPointParameter Name="q6.1" Size="8" Integral="6" Fractional="1"/>
+    <FixedPointParameter Name="q7.0" Size="8" Integral="7" Fractional="0"/>)"};
```

Comment:
BTW, you don't seem to be using any of these parameters (except "nominal") - so you could spare yourself the trouble for now.